README.md

```diff
@@ -5,6 +5,7 @@
 [![CI Django & Postgres Tests](https://github.com/SpaceyaTech/blog/actions/workflows/django-postgres-ci.yml/badge.svg)](https://github.com/SpaceyaTech/blog/actions/workflows/django-postgres-ci.yml)
 
```

Comment:
[![Docker Image CI](https://github.com/SpaceyaTech/mastori/actions/workflows/docker-image.yml/badge.svg)](https://github.com/SpaceyaTech/mastori/actions/workflows/docker-image.yml) 

proposing to add this badge